### ISSUE TITLE
Local .env file

### ISSUE THREAD

**Issue by de-code** [elifesciences/data-pipeline#66]
Create an optional local version of `.env` file to provide environment specific settings that shouldn't be tracked in git.

**Comment by seanwiseman** [comment]
> we could simply set the environment variables we wish to override.

For local dev this would then be done in the `docker-compose.yml` file? would still come with the same risks 🤔

**Comment by seanwiseman** [comment]
I think multiple files would work if the were in gitignore and therefore only created in dev envs.

**Comment by de-code** [comment]
> I think multiple files would work if the were in gitignore and therefore only created in dev envs.

Yes, the idea is that the default file is tracked in git whereas the local version is excluded via .gitignore

> For local dev this would then be done in the docker-compose.yml file?

No, we would set the environment variables outside docker / compose. e.g. one of the below:

```bash
AWS_ACCESS_KEY_ID=abc docker-compose ...
```

```bash
export AWS_ACCESS_KEY_ID=abc
docker-compose ...
```

```bash
source .local-not-tracked-env

docker-compose ...
```

or just put it in `~/.bash_profile` or similar.

In the case of AWS it might actually make sense to actually use `~/.aws/credentials`.
